src/hooks/http.js

```diff
@@ -237,133 +169,42 @@ export const httpRequestArguments = args => {
   return { url, options, callback };
 };
 
-export const getHookedClientRequestArgs = (
-  url,
-  options,
-  callback,
-  requestData,
-  requestRandomId
-) => {
-  const hookedClientRequestArgs = [];
-
-  !!url && hookedClientRequestArgs.push(url);
-
-  if (options) {
-    hookedClientRequestArgs.push(options);
-  }
-
-  if (callback) {
-    const wrappedCallback = exports.wrappedHttpResponseCallback(
-      requestData,
-      callback,
-      requestRandomId
-    );
-    wrappedCallback.__lumigoSentinel = true;
-    hookedClientRequestArgs.push(wrappedCallback);
-  }
+export const httpBeforeRequestWrapper = (args, extenderContext) => {
+  extenderContext.isTracedDisabled = true;
```

Comment:
the comment disappeared,
we wrote here that it is better to change the name of this parameter - it's confusing